examples/ruby/spec/elements/information_spec.rb

```diff
@@ -4,4 +4,48 @@
 
 RSpec.describe 'Element Information' do
   let(:driver) { start_session }
+  let(:url) { 'https://www.selenium.dev/selenium/web/inputs.html' }
+
+  before { driver.get(url) }
+
+  it 'checks if an element is displayed' do
+    displayed_value = driver.find_element(name: 'email_input').displayed?
+    expect(displayed_value).to be_truthy
+  end
+
+  it 'checks if an element is enabled' do
+    enabled_value = driver.find_element(name: 'email_input').enabled?
+    expect(enabled_value).to be_truthy
+  end
+
+  it 'checks if an element is selected' do
+    selected_value = driver.find_element(name: 'email_input').selected?
+    expect(selected_value).to be_falsey
+  end
+
+  it 'gets the tag name of an element' do
+    tag_name = driver.find_element(name: 'email_input').tag_name
+    expect(tag_name).to eq('input')
+  end
+
+  it 'gets the size and position of an element' do
+    size = driver.find_element(name: 'email_input').size
+    expect(size.width).to eq(147)
```

Comment:
Hi @aguspe ,
I think it would be better to verify if value > 0, and not null etc, as the size and position may differ.

I see tests are failing as positions are different!

Reply:
Hi @harsha509, great point, I made the tests more generic now by using be_positive